### ISSUE TITLE
Remove all instances of READTHEDOCS & release 1.0

### ISSUE THREAD

**Issue by ericholscher** [rtfd/sphinx_rtd_theme#650]
We should make this theme like any other Sphinx theme when running on RTD. This means removing all the `{% if READTHEDOCS %}` blocks. There are only a couple left, but we should remove them all.

After we do this and have the theme fully abstracted, I think we should release it as a 1.0. I feel silly having it under 1.0, and want to move to real semvar on our projects going forward.

**Comment by agjohnson** [comment]
I'm fine sticking <1.0 until semver dictates a major version release. We definitely should have started at `1.0` for semver proper, but I don't think that this is a requirement to doing semver successfully. The most important part is that we are communicating backwards compat through versioning, which we are currently. Bumping to `1.0` would be sort of antithetical to these efforts actually.

**Comment by ericholscher** [comment]
1.0 generally means "we'll follow semvar" though -- "Version 1.0.0 defines the public API". So if we're <1.0, it's saying that we can still change whatever we want.

**Comment by agjohnson** [comment]
Agreed we should have started with `1.0`, but we are still following semver. We don't get to change whatever we want without the version reflecting that. I think `<1.0` is more truthful for now, and consistency is more important for package maintenance than correctness is in this case.

**Comment by Blendify** [comment]
@agjohnson was there a reason to remove the 1.0 milestone? I think this would be a nice inclusion in the 1.0 release.

**Comment by agjohnson** [comment]
Mostly because we keep piling things into a 1.0 release and pushing out actually releasing 1.0. This doesn't feel immediately required.

**Comment by agjohnson** [comment]
Also, we're already dropping 1.6 and have some other large changes in our upcoming release. There is a lot of wisdom in cutting more frequent releases, and not making monolithic releases with a lot of important changes combined. This has always felt like the primary thing slowing this project down. 1.1 or great seems like a safe place for this new feature, this doesn't seem important as a 1.0 release feature.

**Comment by benjaoming** [comment]
Pushing this for 2.0 while waiting for more feedback on the PR itself.
